### ISSUE TITLE
Teams erased at programs termination

### ISSUE THREAD

**Issue by NikolasHunter515** [RossNeyman/Team-Building-Project#1]
The employees added to a team, limited to the programs run time. After termination, upon restarting the program the previously created team has been wiped.

**Comment by shauryaCodesAndHosts** [comment]
hi , can i configure the project using maven as i am having problem with dependencies

**Comment by NikolasHunter515** [comment]
Yes, we used maven for our dependencies. Look at the top of the employeeDB file for the dependencies we used. The file is under the util namespace.

**Comment by shauryaCodesAndHosts** [comment]
you want embedded database like H2 or a server based database like postgre ?
